tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -725,4 +738,13 @@ a:after {
 table {
 }
 
+.foo {
+  &-KeepSelector {
+  }
+  &-KeepSelector & .KeepClassSelector {
+  }
+  &-100.200 {
```

Comment:
Is this selector valid?

Reply:
@j-f1 ,i mead 100\.200 it is valid selector